File: android/app/src/main/java/fr/insa/whatodo/models/City.java
package fr.insa.whatodo.models;

/**
 * Created by Benjamin on 31/03/2015.
 */
public class City {
    protected String name;
    protected String ZIPcode;

    public City(String ZIPcode, String name) {
        this.name = name;
        this.ZIPcode = ZIPcode;
    }

    public String getName() {
        return name;
    }

    public String getCode() {
        return ZIPcode;
    }
}

File: web/whatodo/models/city.py
from django.db import models

class City(models.Model):
	id = models.AutoField(primary_key=True)
	
	name = models.CharField(max_length = 50)
	ZIPcode = models.CharField(max_length = 10)
	latitude = models.DecimalField(max_digits=10, decimal_places=7)
	longitude = models.DecimalField(max_digits=10, decimal_places=7)

	def __str__(self):
		return self.name + " (" + self.ZIPcode + ")"

